server/proto/testgen.proto

```diff
@@ -90,13 +90,20 @@ message ProjectContext {
     string buildDirRelativePath = 4;
 }
 
+enum ErrorMode {
+    FAILING = 0;
+    PASSING_IN_TARGET_ONLY = 1;
+    PASSING = 2;
+}
```

Comment:
Could you please add some docs about error mode? Probably, comments or doc at wiki?